imports/plugins/core/shipping/lib/collections/schema/parcelsize.js

```diff
@@ -0,0 +1,36 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { registerSchema } from "@reactioncommerce/reaction-collections";
+
+/**
+ * @name ParcelSize
+ * @memberof schemas
+ * @type {SimpleSchema}
+ * @property {Number} length
+ * @property {Number} width
+ * @property {Number} height
+ * @property {Number} weight
+ */
+export const ParcelSize = new SimpleSchema({
+  length: {
+    type: Number,
+    min: 1,
+    decimal: true
+  },
+  width: {
+    type: Number,
+    min: 1,
+    decimal: true
+  },
+  height: {
+    type: Number,
+    min: 1,
+    decimal: true
+  },
+  weight: {
+    type: Number,
+    min: 1,
```

Comment:
We discussed this and it's unlikely that any other parameter would be >0 but <1 but weight certainly could be, especially if their UOM was kg.